### DISCUSSION TITLE
Metadata formats. csv, json, other or user choice? Balancing human vs computer readable. Discuss.

### DISCUSSION THREAD

**Discussion by regnans** [ess-dive-community/essdive-uas#4] [General]
What are the best format options for metadata? csv, json, other or user choice? How to balance human vs computer readable. Effort required to produce vs functionality. Could format be flexible, depending on data type? What other aspects should be considered?

**Comment by dlebauer** [comment]
> How to balance human vs computer readable.

To collect metadata, it is important to consider how easy (and likely) it is for humans to use. I've had people look at some metadata templates and laugh. Data can be stored in a computer readable format, with interfaces that make it human writable and readable.

**Reply by regnans** [reply]
What do you mean here by metadata template? The possibilities for horrible are quite vast - forms in MS Word format, for example!  
I definitely agree with your last statement, although there are many people who would resist using computer readable because it is unfamiliar to them.

**Reply by bfeddersen327** [reply]
Formats like json or xml can have schemas.  A schema is used to validate your json or xml metadata file.  A schema would not make a good template, but metadata file examples are better to show what the data should look like.
